File: src/002_zlozone_typy_danych/006_typy_plikowe/typy_plikowe_zapis2.py
try:
    with open('dane.txt', 'w') as file:
        # Zapisanie tekstu do pliku
        file.write('To jest przykładowy tekst, który zostanie zapisany do pliku.')
    print('Zapisano dane do pliku.')

except IOError as e:
    print('Błąd zapisu do pliku:', e)
except Exception as ex:
    print('Wystąpił nieoczekiwany błąd:', ex)

File: src/002_zlozone_typy_danych/006_typy_plikowe/typy_plikowe_zapis2.cs
using System;
using System.IO;

class Program
{
    static void Main()
    {
        string fileName = "plik.txt";
        string[] lines = {
            "To jest pierwsza linia tekstu.",
            "To jest druga linia tekstu.",
            "A to jest trzecia linia tekstu."
        };

        try
        {
            using (StreamWriter writer = new StreamWriter(fileName))
            {
                foreach (string line in lines)
                {
                    writer.WriteLine(line);
                }
            }

            Console.WriteLine("Zapisano dane do pliku.");
        }
        catch (Exception ex)
        {
            Console.WriteLine("Błąd zapisu do pliku: " + ex.Message);
        }
    }
}
